.env.local

```diff
@@ -0,0 +1 @@
+REACT_APP_SERVER='https://localhost:4000'
```

Comment:
the use of .local here may be confusing. `.local` has specific meaning in this space. (seems to mean "my local thing that overrides the values as specified in the thing named my name but without .local") Rather, I suggest using `env.localhost` as the filename.

Reply:
I've update the name of the file as you suggested, and the related script as well

Reply:
do we still need this file?